server/gitrest/src/routes/utils.ts

```diff
@@ -18,7 +18,6 @@ export function handleResponse<T>(
             response.status(successStatus).json(result);
         })
         .catch((error) => {
-            winston.error(JSON.stringify(error));
```

Comment:
Was this error line not helpful? Or maybe we are logging the same information elsewhere?

Reply:
This was actually a debug line that accidentally got added and left there in #9136 😬  I wasn't really sure if it was a good idea to generically log an error here, in case I'm missing some knowledge about customer data that could get logged. We should revisit it when we improve logging in GitRest soon